test/puppetlabs/puppetdb/fixtures.clj

```diff
@@ -65,7 +66,9 @@
   is available. Note this means this fixture should be nested _within_
   `with-test-mq`."
   ([f]
-   (binding [*command-app* (command/command-app {:command-mq *mq*})]
+   (binding [*command-app* (command/command-app
+                            {:command-mq *mq*}
+                            #'dispatch/do-enqueue-raw-command)]
```

Comment:
This is passing the `do-enqueue-raw-command` var? I don't understand that, doesn't it just want a function?